src/test/java/integration_tests/ControllerIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package integration_tests;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.KainosJobsApplication;
+import org.kainos.ea.KainosJobsConfiguration;
+import org.kainos.ea.responses.JobRoleResponse;
+
+import javax.ws.rs.client.Client;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class ControllerIntegrationTest {
+
+    private static final DropwizardAppExtension<KainosJobsConfiguration> APP =
+            new DropwizardAppExtension<>(KainosJobsApplication.class);
+
+    @Test
+    void getEmployees_shouldReturnListOfEmployees() {
+        Client client = APP.client();
+
+        List<JobRoleResponse> response = client
+                .target("http://localhost:8080/api/job-roles")
+                .request()
+                .get(List.class);
+
+        Assertions.assertFalse(response.isEmpty());
```

Comment:
why are we asserting is empty?

Reply:
isEmpty checks that the request returns data.
If isEmpty returns false there is data.